contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -930,15 +930,17 @@ def listnodes(self, node_id=None):
         }
         return self.call("listnodes", payload)
 
-    def listpays(self, bolt11=None, payment_hash=None):
+    def listpays(self, bolt11=None, payment_hash=None, status=None):
         """
         Show outgoing payments, regarding {bolt11} or {payment_hash} if set
-        Can only specify one of {bolt11} or {payment_hash}.
+        Can only specify one of {bolt11} or {payment_hash}. It is possible
+        filter the payments by status.
```

Comment:
`status` -> `{status}` to match the formatting of the other params.

Reply:
Thanks for this fix @cdecker and sorry for the delay in fixing it.

Reply:
No worries, thank you for the time you are investing ^^